src/router.zig

```diff
@@ -179,6 +218,12 @@ pub fn router(vrs: *Verse, comptime routes: []const Match) Error!BuildFn {
     return error.Unrouteable;
 }
 
+/// The Verse Server is unlikely to be able to handle the various error states
+/// an endpoint might generate. Pages are permitted to return an error, and the
+/// page builder is required to handle all errors, and make a final decision.
+/// Ideally it should also be able to return a response to the user, but that
+/// implementation detail is left to the caller. This default builder is provide
```

Comment:
The last sentence doesn't make sense, I think maybe you meant this:

This default build is provided and can handle an abbreviated number of errors.